src/pages/Swap/LimitOrderBox/components/OrderLimitPriceField/OrderLimitPriceField.tsx

```diff
@@ -200,3 +182,33 @@ export function OrderLimitPriceField({ id }: OrderLimitPriceFieldProps) {
     </InputGroup>
   )
 }
+
+export function calcualtePriceDiffPercentage(
+  limitOrder: SerializableLimitOrder,
```

Comment:
Maybe name this to `calculateMarketPriceDiffPercentage` to be more clear and no need to pass all limitOrder here, passing `limitOrderKind` is enough